plugins/modules/foreman_host.py

```diff
@@ -50,18 +51,29 @@
     description:
       - Name of related organization
     required: false
+  build:
+    description:
+      - Whether or not to setup build context for the host
+    type: bool
+    required: false
+    default: None
   enabled:
     description:
       - Include this host within Foreman reporting
     type: bool
     required: false
-    default: true
+    default: None
   managed:
     description:
-      - whether a host is managed or unmanaged
+      - Whether a host is managed or unmanaged.
+      - Forced to true when I(build=true)
     type: bool
     required: false
     default: None
```

Comment:
this conflicts after I've merged https://github.com/theforeman/foreman-ansible-modules/pull/433 -- sorry.

Reply:
merged
and added docstring for state